Example/Custom Integration (ObjC)/BrowseExamplesViewController.m

```diff
@@ -123,7 +133,69 @@ - (void)createBackendChargeWithSource:(NSString *)sourceID completion:(STPSource
                                                           if (error) {
                                                               completion(STPBackendResultFailure, error);
                                                           } else {
-                                                              completion(STPBackendChargeResultSuccess, nil);
+                                                              completion(STPBackendResultSuccess, nil);
+                                                          }
+                                                      }];
+
+    [uploadTask resume];
```

Comment:
why was this added?

Reply:
This is pre-existing code, it's just the Github diff algorithm is being weird. Both methods end the same way (L202 has the same call).